### DISCUSSION TITLE
YT-dlp OAuth plugin support?

### DISCUSSION THREAD

**Discussion by eheupel** [jmbannon/ytdl-sub#1040] [Ideas]
Is it possible to add (or point me how to enable) support for the yt-dlp Oauth plugin ( https://github.com/coletdjnz/yt-dlp-youtube-oauth2 ). Without it running into blocks from Youtube, but when I used it on the yt-dlp command inside my vm it worked a charm. Unfortunately I could not see how to get ytdl-sub to pass it or to add it to the yt-dlp config file with it being accepted when called by ytdl-sub. 

Other than this new issue from google/youtube, I am seriously loving ytdl-sub. Thank you!!!

**Comment by jmbannon** [comment]
yt-dlp is installed in the docker container using pip, so the install instructions on their readme I think should work. Just run yt-dlp once to enter the code, then it should work for sub
